sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCSuite.scala

```diff
@@ -685,6 +699,17 @@ class JDBCSuite extends QueryTest
     JdbcDialects.unregisterDialect(testH2Dialect)
   }
 
+  test("Map TINYINT to ByteType via JdbcDialects") {
```

Comment:
We are trying to normalize this officially.
```scala
- test("Map TINYINT to ByteType via JdbcDialects") {
+ test("SPARK-26499: Map TINYINT to ByteType via JdbcDialects") {
```

- https://github.com/apache/spark-website/pull/231